README.md

```diff
@@ -194,6 +193,21 @@ Alternatively, link against the `benchmark_main` library and remove
 The compiled executable will run all benchmarks by default. Pass the `--help`
 flag for option information or see the guide below.
 
+#### Usage with CMake
+If using CMake, it is recommended to link against the project-provided `benchmark::benchmark` and `benchmark::benchmark_main` targets using `target_link_libraries`.
+It is possible to use ```find_package``` to import an installed version of the library.
+```cmake
+find_package(benchmark REQUIRED)
+```
+Alternatively, ```add_subdirectory``` will incorporate the library directly in to one's CMake project.
+```cmake
+add_subdirectory(benchmark)
+```
+Either way, link to the library as follows.
+```cmake
+target_link_libraries(MyTarget benchmark::benchmark)
+```
+
```

Comment:
1. Newline after the header
2. I think some docs lines aren't properly linewrapped

Reply:
@LebedevRI I have added a newline after the header, ensured the lines I modified wrapped to a maximum length of 80 characters, and also moved the *Usage with CMake* up one header-level to make it a direct descendant of the *Usage* section.